packages/legacy/jest.config.js

```diff
@@ -0,0 +1,54 @@
+const { pathsToModuleNameMapper } = require('ts-jest/utils');
+const ts = require('typescript');
+
+function readTsConfig(path = "./") {
+  const parseConfigHost = {
+    fileExists: ts.sys.fileExists,
+    readFile: ts.sys.readFile,
+    readDirectory: ts.sys.readDirectory,
+    useCaseSensitiveFileNames: true
+  };
+
+  const configFileName = ts.findConfigFile(
+    path,
+    ts.sys.fileExists,
+    "tsconfig.json"
+  );
+  const configFile = ts.readConfigFile(configFileName, ts.sys.readFile);
+  const compilerOptions = ts.parseJsonConfigFileContent(
+    configFile.config,
+    parseConfigHost,
+    path
+  );
+  return compilerOptions;
+}
+
+const { options } = readTsConfig();
+
+module.exports = {
+  roots: ['<rootDir>/src'],
+  /**
+   *  If you comment this out, you will get error unexpected token with optional chaining because you are using babel in your project
+   *  When using babel together with ts-jest in a project, you need to let ts-jest know about it
+   */
+  globals: {
```

Comment:
can you put this comment in the other place you do this

Reply:
already in the other jest config